mirrord/cli/src/config.rs

```diff
@@ -223,6 +223,39 @@ pub(super) enum OperatorCommand {
         #[arg(short = 'f')]
         config_file: Option<String>,
     },
+    /// Operator session management commands.
+    ///
+    /// Allows the user to forcefully kill living sessions.
+    #[command(subcommand)]
+    Session(SessionCommand),
+}
+
+/// `mirrord operator session` family of commands.
+///
+/// Allows the user to forcefully kill operator sessions, use with care!
+#[derive(Debug, Subcommand, Clone, Copy)]
+pub(crate) enum SessionCommand {
+    /// Kills the session specified by `id`.
+    Kill {
+        /// Id of the session.
+        #[arg(short, long, value_parser=hex_id)]
+        id: u64,
+    },
+    /// Kills all operator sessions.
+    KillAll,
+
+    /// Kills _inactive_ sessions, might be useful if an undead session is still being stored in
+    /// the session storage.
+    #[clap(hide(true))]
+    RetainActive,
+}
+
+/// Parses the operator session id from hex (without `0x` prefix) into `u64`. It also accepts the
+/// non-hex version of the id.
+fn hex_id(raw: &str) -> Result<u64, String> {
```

Comment:
Session id is generated randomly in `mirrord_operator` crate. What if its hex representation is also a valid decimal representation? :X
E.g. the user could pass `0x1111` id as `1111` (because we omit the `0x` prefix) and it would parse to a different value.
`mirrord operator status` always returns session ids in hex, so I think it would be better to parse only hex values

Reply:
Only hex now, good catch.